extensions/ql-vscode/src/codeql-cli/distribution.ts

```diff
@@ -587,10 +631,9 @@ export class ReleasesApiConsumer {
     });
     // Tag names must all be parsable to semvers due to the previous filtering step.
     const latestRelease = compatibleReleases.sort((a, b) => {
-      const versionComparison = semver.compare(
-        semver.parse(b.tag_name)!,
-        semver.parse(a.tag_name)!,
-      );
+      const versionComparison = orderBySemver
+        ? semver.compare(semver.parse(b.tag_name)!, semver.parse(a.tag_name)!)
+        : b.id - a.id;
```

Comment:
I wasn't sure about whether to order based on `id` or `createdAt`, but went with `id` because it's a number and having to parse date strings (which might be invalid) so we can compare them is a pain.

I also wondered about passing in a comparison function instead this `orderBySemver` boolean, but thought maybe that could wait for followup refactoring.